src/sage/rings/polynomial/polynomial_ring_constructor.py

```diff
@@ -716,9 +720,15 @@ def PolynomialRing(base_ring, *args, **kwds):
             raise TypeError("variable names specified twice inconsistently: %r and %r" % (names, kwnames))
 
     if multivariate or len(names) != 1:
-        return _multi_variate(base_ring, names, **kwds)
+        if isinstance(base_ring, TropicalSemiring):
+            return TropicalMPolynomialSemiring(base_ring, len(names), names, **kwds)
+        else:
+            return _multi_variate(base_ring, names, **kwds)
```

Comment:
```suggestion
        return _multi_variate(base_ring, names, **kwds)
```
Trivial `else` statements after a `return` are annoying IMO.